test/lexer/Lexer.test.js

```diff
@@ -341,4 +341,38 @@ describe("lexer", () => {
             ]);
         });
     });
+
+    describe('two word keywords', () => {
+        it('supports various spacing between for each', () => {
+            var k = 2;
+            let { tokens } = Lexer.scan('for each for  each for    each for\teach for\t each for \teach for \t each');
+            console.log(tokens);
```

Comment:
We probably don't need this anymore :wink:

Reply:
removed!

Reply:
Hmm, perhaps my browser's borked - I control-shift-R'd and it's still there! :thinking:

Reply:
Yeah, it might be messed up. Switch to the "Files changed" tab and see if the line still exists over there. I don't see it over there...but the comment above is definitely still showing it.

Reply:
Oh, I didn't remove the console.log. Der.